cms/djangoapps/contentstore/tasks.py

```diff
@@ -737,14 +739,18 @@ def validate_course_olx(courselike_key, course_dir, status):
     if not course_import_olx_validation_is_enabled():
         return olx_is_valid
     try:
+        if str(courselike_key) == "course-v1:ArbiX+CS101+2014_T3":
+            logging.info(log_prefix + "Validating. Calling olxcleaner.validate")
         __, errorstore, __ = olxcleaner.validate(
             filename=course_dir,
             steps=settings.COURSE_OLX_VALIDATION_STAGE,
             ignore=settings.COURSE_OLX_VALIDATION_IGNORE_LIST,
             allowed_xblocks=ALL_ALLOWED_XBLOCKS
         )
-    except Exception:  # pylint: disable=broad-except
-        LOGGER.exception(f'{log_prefix}: CourseOlx could not be validated')
+        if str(courselike_key) == "course-v1:ArbiX+CS101+2014_T3":
+            logging.info(log_prefix + "Course validated. No errors")
+    except Exception as e:  # pylint: disable=broad-except
+        LOGGER.exception(f'{log_prefix}: CourseOlx could not be validated' + str(e))
```

Comment:
`LOGGER.exception` automatically prints the stack trace so I don't think `str(e) is needed here.